sumcheck/src/prover/high_degree.rs

```diff
@@ -311,6 +331,10 @@ fn sumcheck_round<E: FieldElement>(
                 let eq_at_zero = eq_ml.evaluations()[2 * i];
                 let eq_at_one = eq_ml.evaluations()[2 * i + 1];
 
+                // add evaluation of periodic columns
+                periodic_table.get_periodic_values_at(2 * i, &mut evals_zero[num_mls..]);
+                periodic_table.get_periodic_values_at(2 * i + 1, &mut evals_one[num_mls..]);
```

Comment:
I don't think we can assume that periodic values will always be at the end of a `query`, right?